packages/vue-cli-plugin-vuetify/generator/tools/alaCarte.js

```diff
@@ -16,6 +16,18 @@ function addDependencies (api, useV3) {
   api.extendPackage(deps)
 }
 
+function addVueConfigPluginOptions(api) {
+  api.extendPackage({
+    vue: {
+      pluginOptions: {
+        vuetify: {
+          styles: 'expose',
```

Comment:
Do we want this to be the default?

Reply:
No, the default should be to load the component's pre-compiled CSS file.